sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallConnectionAsyncLiveTests.java

```diff
@@ -2,30 +2,34 @@
 // Licensed under the MIT License.
 package com.azure.communication.callingserver;
 
-import java.util.UUID;
-
-import com.azure.communication.callingserver.models.CallModality;
+import com.azure.communication.callingserver.models.AddParticipantResult;
 import com.azure.communication.callingserver.models.CancelAllMediaOperationsResult;
-import com.azure.communication.callingserver.models.EventSubscriptionType;
 import com.azure.communication.callingserver.models.CreateCallOptions;
+import com.azure.communication.callingserver.models.EventSubscriptionType;
 import com.azure.communication.callingserver.models.JoinCallOptions;
+import com.azure.communication.callingserver.models.MediaType;
 import com.azure.communication.callingserver.models.PlayAudioResult;
 import com.azure.communication.common.CommunicationIdentifier;
 import com.azure.communication.common.CommunicationUserIdentifier;
 import com.azure.communication.common.PhoneNumberIdentifier;
 import com.azure.core.http.HttpClient;
 import com.azure.core.http.rest.Response;
-
+import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
+import java.util.UUID;
+
 public class CallConnectionAsyncLiveTests extends CallingServerTestBase {
 
     private final String fromUser = getNewUserId();
     private final String toUser = getNewUserId();
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "COMMUNICATION_SKIP_INT_CALLINGSERVER_TEST",
+        matches = "(?i)(true)")
```

Comment:
FYI: It is also possible to add a comment for reason, disabledReason="Require human intervention"

Reply:
disable reason added

    @EnabledIfEnvironmentVariable(
        named = "RUN_CALLINGSERVER_TEST_RECORD",
        matches = "(?i)(true)",
        disabledReason = "Requires human intervention")